tempto-core/src/test/resources/configuration/global-configuration-tempto.yaml

```diff
@@ -2,6 +2,7 @@ value:
   default: ${not.exisiting.key:-default_value}
   both: global
   global: global
+  with.dot: 1
```

Comment:
is this valid yaml file?

Reply:
Good point. The valid syntax is `"with.dot":`